lib/rules/web/admin_console/4.2/common_ui_elements.xyaml

```diff
@@ -73,6 +74,26 @@ set_input_value:
       xpath: //input[@id='<input_field_id>']
     op: send_keys <input_value>
     type: input
+# click and check the first item from resource list such as projects, pods, etc.
+click_first_item_from_resource_list_and_check_breadcrumb:
+  action: click_first_item_in_resource_list
+  action: check_link_in_breadcrumb
+# click and check the first sub item from a resource such as one of imagestreamtag on imagestream detail page
+click_first_item_from_sub_resource_list_and_check_breadcrumb:
+  action: click_first_sub_item_in_resource_detail_list
+  action: check_link_in_breadcrumb
+# click and check the first item from resources listed in grid cell, such as installed operator
+click_first_item_from_gridcell_list_and_check_breadcrumb:
+  action: click_first_item_in_grid_cell_list
+  action: check_link_in_breadcrumb
+check_link_in_breadcrumb:
+  element:
+    selector:
+      xpath: //ol[contains(@class, 'breadcrumb')]/li[<layer_number>]/a[contains(@data-test-id,'breadcrumb-link') and contains(@href,'<link>') and translate(normalize-space(text()), ' ','')='<text>']
```

Comment:
4.2 and 4.3 has different text in breadcrumb. 4.2 is `Install Plan` and 4.3 is `InstallPlan` without any white space. 
Use `translate(normalize-space(text()), ' ','')='<text>'` here to match `Multiple Words` in unique format of `MultipleWords`, feature params are updated as well.